spec/unit/services_inspector_spec.rb

```diff
@@ -111,12 +111,47 @@
   start on startup (job:, env:)
 EOF
 
+      service_ubuntu_output =
+        <<-EOF
+ [ + ]  apparmor
+ [ + ]  ntp
+ [ - ]  ssh
+ [ - ]  rsync
+EOF
+
+      service_ubuntu_error =
+        <<-EOF
+ [ ? ]  console-setup
+EOF
+
       allow(system).to receive(:has_command?).
         with("systemctl").and_return(false)
       allow(system).to receive(:has_command?).
         with("initctl").and_return(true)
       expect(system).to receive(:run_command).
         with(
+          "/usr/sbin/service",
+          "--status-all",
+          :stdout=>:capture, :stderr=>:capture
+        ).
+        and_return([service_ubuntu_output,service_ubuntu_error])
+
+      [2,3,4,5,"S"].map do |runlevel|
+        expect(system).to receive(:run_command).
+          with(
+            "/usr/bin/find",
+            "/etc/rc#{runlevel}.d",
+            "-name",
+            "S*",
+            {:stdout=>:capture}
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.